templates/nav.html

```diff
@@ -28,6 +28,11 @@
                     <a href="/users">User Management<span class="nav-badge badge pull-right">Admin</span></a>
                 </li>
                 {{end}}
+                {{if .ModifySystem}}
```

Comment:
Since "User Management` is already wrapped in an `if .ModifySystem`, we can remove this and just move the `li` into the block above. So it'd look like:

```
{{if .ModifySystem}}
<li>
    <a href="/users">User Management<span class="nav-badge badge pull-right">Admin</span></a>
</li>
<li>
    <a href="/webhooks">Webhooks<span class="nav-badge badge pull-right">Admin</span></a>
</li>
{{end}}
```